bindings/go/examples/sign_send_iota/main.go

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"log"
+
+	sdk "bindings/iota_sdk_ffi"
+)
+
+func main() {
+	// Amount to send in nanos
+	amount := uint64(1000)
+	recipientAddress, err := sdk.AddressFromHex("0x0000a4984bd495d4346fa208ddff4f5d5e5ad48c21dec631ddebc99809f16900")
+	if err != nil {
+		log.Fatalf("Failed to parse recipient address: %v", err)
+	}
```

Comment:
I dunno but to keep the example less cluttered, I would just assume that this can never fail

Reply:
Assert where?

Reply:
I am just wondering whether we can remove all those redunandent error handling code in all the examples when we know that the parsing cannot fail, because the focus of those examples is not to show correct error handling IMHO, but to show how to `sign + send iota` in this case. Why have this noise in the example? I think people who are interested in the Go example already know how error handling works.

Reply:
okay, let's keep it .... those Go errors ... wtf